distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1435,7 +1433,7 @@ public void testVideoThumbnail() throws Exception {
         try (InputStream inputStream = IOUtils.toInputStream(getFileContent("sample.mp4"))) {
             final byte[] fileBytes = IOUtils.toByteArray(inputStream);
 
-            final ValidatableResponse response = given().multiPart("file",
+            given().multiPart("file",
                     "sample.mp4",
```

Comment:
✏️ Looks like we should have been using that variable defined above instead of inlining it here.